profile.d/go.sh

```diff
@@ -0,0 +1,4 @@
+# oh my go.sh
+# https://github.com/golang/go/wiki/GOPATH
+export GOPATH="$HOME/go"
```

Comment:
1. Using `~/go` for Go's internal source store is kinda dirty/confusing.  Why not `~/.cache/gocode` or `~/.local/gocode`?
2. This is masking system-wide Go packages that live in `/usr/share/gocode`. Is it on purpose?